sdk/samples/radio_example_rx/main.c

```diff
@@ -7,19 +7,44 @@
 #include "radio.h"
 #include "rftimer.h"
 #include "scm3c_hw_interface.h"
+#include "scum.h"
 
-// Number of for loop cycles between Hello World messages.
-// 700000 for loop cycles roughly correspond to 1 second.
-#define NUM_CYCLES_BETWEEN_TX (1000000UL)
+uint8_t packet[125] = {0};
+uint8_t packet_len = 125;
+int8_t rxpk_rssi = 0;
+uint8_t rxpk_lqi = 0;
```

Comment:
Should all be declared static:
```suggestion
static uint8_t packet[125] = {0};
static uint8_t packet_len = 125;
static int8_t rxpk_rssi = 0;
static uint8_t rxpk_lqi = 0;
```